### PR TITLE
feat(website): redo whitespaces demo page

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Updated the `Whitespaces` section of plasma website. Also, I renamed it `Spacing`

### Potential Breaking Changes

None

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/website/src/pages/foundations/Spacing.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+
+import {PageLayout} from '../../building-blocs/PageLayout';
+
+const code = `
+    import * as React from 'react';
+
+    // red is margin
+    // blue is padding
```
Comment (GermainBergeron) [inline packages/website/src/pages/foundations/Spacing.tsx:9]: I would either add a comment for `highlight-padding` and `highlight-margin` to say they are only for the demo or add a way to specify a class for the sandbox container

```suggestion
    // red is margin
    // blue is padding
    // highlight-padding and highlight-margin are only available on the demo website
```

**Review by GermainBergeron — APPROVED** [review]
Really nice!

**Comment by aiheon** [comment]
When I click on the tile from the home page, it does not navigate event if the path is set in the url :raised_eyebrow:  ...

**Comment by aiheon** [comment]
j'ai encore des access denied :confused: 
image

**Review by toofff — APPROVED** [review]

**Pull request merged**